riff-raff.yaml

```diff
@@ -15,7 +15,7 @@ deployments:
     parameters:
       amiParameter: AMIAmiable
       amiTags:
-        Recipe: arm64-bionic-java8-deploy-infrastructure
+        Recipe: arm64-bionic-java8-deploy-infrastructure-test
```

Comment:
Is this meant to be reverted prior to deploy or needed once merged?

Reply:
Thanks for catching this - I will release `devx-logs`, bake a new `arm64-bionic-java8-deploy-infrastructure` AMI, then revert this change before merging